content/en/Platform Deep Dive/Cobalt Account/account-recovery.md

```diff
@@ -99,6 +100,28 @@ To reset your password:
 1. Enter your email address that you used to [sign in to Cobalt](/getting-started/sign-in/), and select **Reset Password**.
 1. Follow the instructions in the email you receive.
 
+## We Can't Recognize Your Device
+
+{{% device-verification-intro %}} This may happen when you:
+
+- Sign in from a new browser
+- Use your browser's private (incognito) mode
+- Clear your site data
+- Sign in from a different system
+
+When you see the **Verify It's You** message upon signing in, do the following:
+
+1. Check your email that you used to sign in to Cobalt.
+1. In the email prompting you to verify your device, double-check the details of your last sign-in attempt, and select **Verify Device**.
```

Comment:
@molfinn Should we mention that the user needs to click Verify Device within 5 minutes (while the token is valid)?
 I assume the following scenarios are edge cases:
- The user deletes the email.
- The user clicks Verify Device when the token expired.

Reply:
I would assume those 2 bullets are edge cases, yes. Do you mention in other locations the need to take action before expires? The email does not include any messaging on expiration (may want to add). I would say yes, it would be good to mention.

Reply:
@molfinn Added this:

> - Make sure to verify your device before the link in the email expires.